DropcastDev/DropcastDev.xcodeproj/project.pbxproj

```diff
@@ -152,7 +152,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which mint > /dev/null; then\n  xcrun --sdk macosx mint run swiftlint lint --fix $(git rev-parse --show-toplevel)\nelse\n  echo \"warning: mint not installed\"\nfi\n";
+			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which mint > /dev/null; then\n  git diff --name-only | grep \".swift$\" | while read filename; do\n    xcrun --sdk macosx mint run swiftlint lint --fix --path $filename\n  done\nelse\n  echo \"warning: mint not installed\"\nfi\n";
```

Comment:
In fact, the build failed with the original script. [ref](https://github.com/konifar/Dropcast/actions/runs/4155447639/jobs/7188536561)

I didn't know what's happened but I guessed this script can be improved.
It would be faster to check only `.swift` files which are output by `git diff` instead of all `.swift` files.

Reply:
Thanks! I found two points on this diff that we can improve.

1. A warning below is generated on build phase. This should be fixed by just dropping `--path`.

<img width="664" alt="image" src="https://user-images.githubusercontent.com/22269397/218345309-2b2e24cd-a8d4-45f0-9781-1dd43730ecc4.png">

2. When swift files outside the project become lint target, an error occurred because the project cannot find the file. For example, the screenshot below shows the error output when I edit `Package.swift`. This is crucial because most of the files exist outside project (= in `DropcastPackage`) in this repository.

<img width="765" alt="image" src="https://user-images.githubusercontent.com/22269397/218345535-865f95ff-66dc-49a3-804c-64171d38caa8.png">

Maybe we can fix this problem by passing filenames to swiftlint by full path.

ref: https://stackoverflow.com/questions/54697900/git-diff-name-only-with-full-file-path

I think the above two fixes can be summarized as follows. What do you think?

```suggestion
			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which mint > /dev/null; then\n  git diff --name-only --line-prefix=$(git rev-parse --show-toplevel)/ | grep \".swift$\" | while read filename; do\n    xcrun --sdk macosx mint run swiftlint lint --fix $filename\n  done\nelse\n  echo \"warning: mint not installed\"\nfi\n";
```

Reply:
Thanks! That would be better. I merged your suggestion!